pkg/cmd/generate/util.go

```diff
@@ -158,8 +159,11 @@ const header = `# --------------------------------------------------------------
 
 func writeFile(filePath string, content []byte) error {
 	// https://github.com/kubernetes/kubernetes/issues/67610
-	contentString := strings.ReplaceAll(string(content), "\n  creationTimestamp: null", "")
+	emptyCreationTimestamp := regexp.MustCompile(`\n *creationTimestamp: null`)
+	contentString := emptyCreationTimestamp.ReplaceAllString(string(content), "")
 	contentString = strings.ReplaceAll(contentString, "\nuser: {}", "")
+	// This will only apply to konflux tier template that start with metadata = {}
+	contentString = strings.ReplaceAll(contentString, "metadata:\n    objects:", "metadata: {}\n    objects:")
```

Comment:
This fixes the metadata problem with Konflux as tiers in src already start with spec.template.metadata as {}. See here: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/sandbox/tiers/src/appstudio/spacerole_admin.yaml#L3
But this won't affect our e2e nor unitest as those start with spec.template.metadata.name having a string value. Would this work?

Reply:
looks good, this should work fine :+1: :rocket: